ArcadiaTeamsBot/Dialogs/NewRequestDialog.cs

```diff
@@ -1,63 +1,288 @@
 ﻿namespace ArcadiaTeamsBot.Dialogs
 {
+    using System;
     using System.Collections.Generic;
+    using System.Linq;
     using System.Threading;
     using System.Threading.Tasks;
 
+    using AdaptiveCards;
+
+    using ArcadiaTeamsBot.CQRS.Abstractions;
+    using ArcadiaTeamsBot.CQRS.Abstractions.Commands;
+    using ArcadiaTeamsBot.ServiceDesk.Abstractions;
+    using ArcadiaTeamsBot.ServiceDesk.Abstractions.DTOs;
+    using ArcadiaTeamsBot.ServiceDesk.Requests.RequestType;
+    using ArcadiaTeamsBot.ServiceDesk.Requests.RequestTypeFactory;
+
+    using MediatR;
+
     using Microsoft.Bot.Builder;
     using Microsoft.Bot.Builder.Dialogs;
     using Microsoft.Bot.Schema;
 
-    public class NewRequestDialog : ComponentDialog
+    using Newtonsoft.Json.Linq;
+
+    internal class NewRequestDialog : ComponentDialog
     {
-        private const string Back = "Back";
+        private const string Submit = "Submit";
+        private const string Cancel = "Cancel";
+        private const string Yes = "Yes";
+        private const string No = "No";
+        private const string Button = "Button";
+        private const string Title = "Title";
+        private const string TypeId = "TypeId";
+        private const string InputValidation = "InputValidation";
+        private const string Priority = "Priority";
+        private const string Description = "Description";
+        private const string ExecutionDate = "Execution Date";
+        private const string Username = "ekaterina.kuznetsova@arcadia.spb.ru";
+        private readonly IRequestTypeUIFactory requestTypeUIFactory;
```

Comment:
Please, add empty line between constants and fields.

Reply:
Added